src/shared/components/MedicalDisclaimer.tsx

```diff
@@ -1,8 +1,8 @@
 import { HeroContainer } from 'franklin-sites';
 
-const MedicalDisclaimer = ({ title = <h3>Disclaimer</h3> }) => (
+const MedicalDisclaimer = () => (
   <HeroContainer>
-    {title}
+    <h2 className="medium">Disclaimer</h2>
```

Comment:
Changed now.
Note that I had changed the name of the prop in Franklin because `title` is already something with another meaning on a HTML element (accessible title with native small tooltip), so apart from the possible misuse of it, it would prevent spreading props to the underlying HTML element if we ever wanted an actual native `title` on this component.